demo-project/pom.xml

```diff
@@ -1,153 +1,160 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
You generally shouldn't include line changes that only change formatting, unless the intent of the commit is a change in formatting style. Changes like those make it very hard to see what functional changes the commit contains.